Intersect.Client.Core/Interface/Game/Inventory/InventoryWindow.cs

```diff
@@ -97,21 +97,13 @@ public override void Hide()
         }
 
         _contextMenu?.Close();
-        base.Hide();
-    }
 
-    // TODO: Window has RenderBounds as property, but InventoryWindow has it as a method. This should be consistent.
-    public FloatRect RenderBounds()
-    {
-        var tempSlot = Items[0];
-        var rect = new FloatRect()
+        // dont hide window if we are dragging something
+        if (Items.Any(c => c.IconImage == DragAndDrop.CurrentPackage?.DrawControl))
```

Comment:
Same comment as in `BagWindow`

Also since this is repeated at least 4 times I think you should add a `// TODO: Create a base 'SlotWindow' that BagWindow/BankWindow/InventoryWindow/SpellsWindow should inherit from so this code can be written only once`

Reply:
It is no longer repeated 4 times so I will not add the comment, the correct thing would be to block at the root, and I tried to do that now, the windows do not hide and others do not open while dragging, which is correct